server.go

```diff
@@ -768,6 +783,31 @@ func (s *Server) InlineParseCustomData(msg proto.Message) error {
 	return cmsg.ParseCustomChannelData(msg)
 }
 
+// Name returns the name of this endpoint. This MUST be unique across all
+// registered endpoints.
+//
+// NOTE: This method is part of the protofsm.MsgEndpoint interface.
+func (s *Server) Name() protofsm.EndpointName {
+	return tapchannel.MsgEndpointName
```

Comment:
The return value here doesn't seem appropriate to me. `Server` is the main taproot asset server but `tapchannel.MsgEndpointName` is specific to channel funding: "taproot assets channel funding".

Reply:
As discussed offline, the whole of `tapd` is just a single message endpoint. Ideally we'd want this value to be returned from the `AuxFundingController`. But that instance isn't ready to be called into at the moment we get called here, so we just return the same constant as the `AuxFundingController` also would.